### ISSUE TITLE
[BUG] pt: virial training seems unsupported

### ISSUE THREAD

**Issue by njzjz** [deepmodeling/deepmd-kit#3482]
### Bug summary

No virial loss is outputted even when `start_pref_v` and `limit_pref_v` are changed to non-zero values.

### DeePMD-kit Version

v3.0.0a0-28-ged831c88-dirty

### TensorFlow Version

PT v2.2.0+cu121-g8ac9b20d4b0

### How did you download the software?

From source

### Input Files, Running Commands, Error Log, etc.

`lcurve.out` doesn't show anything for virial.

```
#  step      rmse_val    rmse_trn    rmse_e_val  rmse_e_trn    rmse_f_val  rmse_f_trn         lr
      0      2.76e+01    2.61e+01      5.67e-01    9.28e-01      8.72e-01    8.25e-01    1.0e-03
    100      9.46e+00    9.62e+00      1.87e-01    2.06e-01      2.99e-01    3.04e-01    1.0e-03
```

### Steps to Reproduce

`start_pref_v` and `limit_pref_v` are changed to 
```diff
diff --git a/examples/water/se_e2_a/input_torch.json b/examples/water/se_e2_a/input_torch.json
index fe424afe..b41cd35b 100644
--- a/examples/water/se_e2_a/input_torch.json
+++ b/examples/water/se_e2_a/input_torch.json
@@ -49,6 +49,8 @@
     "limit_pref_e": 1,
     "start_pref_f": 1000,
     "limit_pref_f": 1,
+    "start_pref_v": 1,
+    "limit_pref_v": 1,
     "_comment": " that's all"
   },
   "training": {
```

Then run
```sh
cd examples/water/se_e2_a
dp --pt train input_torch.json
```

**Comment by njzjz** [comment]
This issue seems to be false positive...
